src/main/resources/db/migration/v0.37.0.65__gaf-load.sql

```diff
@@ -0,0 +1,204 @@
+insert into bulkloadgroup (id, name, internal, obsolete, dbdatecreated)
```

Comment:
FYI, my PR went in with the same migration version, the migration version of this might need to be changed to 37.0.66

Reply:
Thanks for the reminder.